src/GitTools.IssueTrackers/IssueTrackers/IssueTrackerBase.cs

```diff
@@ -1,18 +0,0 @@
-﻿namespace GitTools.IssueTrackers
-{
-    using System;
-    using System.Collections.Generic;
-    using System.Threading.Tasks;
-
-    public abstract class IssueTrackerBase : IIssueTracker
```

Comment:
For now it might seem purposeless, but allows us to add more stuff the the base class in the future without breaking changes.